RELEASES.md

```diff
@@ -1,3 +1,105 @@
+Version 1.30.0 (2018-10-25)
+==========================
+
+Language
+--------
+- [This version marks the release of the 2018 edition of Rust.][54057]
+- [In the 2018 edition you can now use `crate` visibility keyword.][54057]
+  Equivalent to `pub(crate)`.
+- [You can now use keywords as identifiers using the raw identifiers
+  syntax (`r#`).][53236] e.g. `let r#struct = 2018;`
+- [Using anonymous parameters in traits is now deprecated with a warning and is
+  a hard error in the 2018 edition.][53272]
+- [You can now apply the `#[used]` attribute to static items to prevent the
+  compiler from optimising them away even if they appear to be unused.][51363]
+  e.g. `#[used] static FOO: u32 = 1;`
+- [You can now import and reexport macros using the `use` syntax.][50911]
+- [`mod.rs` files are now optional, you can now have `src/foo.rs` and
+  `src/foo/bar.rs` to define module dependencies.][54146]
+- [`rustc` can now infer certain outlives requirements.][53793] e.g. In this
+  example `struct Foo<'a, T: 'a>(&'a T);` you now no longer need the `T: 'a`
+  requirement for the code to work, it will now be inferred.
+- [You can now catch visibility keywords (e.g. `pub`, `pub(crate)`) in macros
+  using the `vis` specifier.][53370]
+- [Attributes now allow all forms of literals not just strings.][53044] e.g.
+  Previously you would write `#[attr("true")]` you can now
+  write `#[attr(true)]`.
+- [You can now specify a function to handle a panic in the Rust runtime with the
+  `#[panic_handler]` attribute.][51366]
+
+Compiler
+--------
+- [Added the `riscv32imc-unknown-none-elf` target.][53822]
+- [Added the `aarch64-unknown-netbsd` target][53165]
+
+Libraries
+---------
+- [`ManuallyDrop` now allows the inner type to be unsized.][53033]
+
+Stabilized APIs
+---------------
```

Comment:
Proc macro API was stabilized in 1.29 (https://github.com/rust-lang/rust/pull/52081), but never mentioned in release notes.
In 1.30 there's no reason to be secretive about it, so it can be announced.